main.go

```diff
@@ -206,6 +169,51 @@ func run(c *cli.Context) error {
 		secretTemplate = ".kube.sec.yml"
 	}
 
+	// Parse variables.
+	vars := make(map[string]interface{})
+	varsJSON := c.String("vars")
+	if varsJSON != "" {
+		if err := json.Unmarshal([]byte(varsJSON), &vars); err != nil {
+			return fmt.Errorf("Error parsing vars: %s\n", err)
+		}
+	}
+
+	// Parse secrets.
+	secrets := make(map[string]string)
+	for _, e := range os.Environ() {
+		if !strings.HasPrefix(e, "SECRET_") {
```

Comment:
Let's talk about this for a second.

I feel like it is a little convoluted to force the user to remap secret names...

```yaml
    secrets:
      - source: API_TOKEN
        target: secret_api_token
```

... and then use their uppercased version in their template ...

```yaml
data:
  api-token: {{.SECRET_API_TOKEN}}
```

... because it's not really expected.

Unless https://github.com/drone/drone/issues/2088 is implemented, we can't automatically parse this out.

I don't want to add another field specifying the "list" of secrets being used, since that almost seems like the equivalent of `target:`.

Ideally it would look something like this:

```yaml
    secrets: [API_TOKEN, lowercase_key_ok]
```

```yaml
data:
  api-token: {{.API_TOKEN}}
  my-secret: {{.lowercase_key_ok}}
```

Note: to achieve `lowercase_key_ok` (key case sensitivity) would require https://github.com/drone/drone/issues/2070 as well.